### ISSUE TITLE
Update on next mod patch? 

### ISSUE THREAD

**Issue by ItsG4T0R** [VoidArkMods/QoLPlus#188]
Hello,

     There are quite a few issues with the QOL mod that are pending fixes and am curious if you have any timeline on when this patch will be pushed out?

**Comment by VoidArkMods** [comment]
Hello.
The next patch will most likely contain some stability enhancements as well as the fix for the propagator that can't be interacted with on dedicated servers. The issue is: my computer is in an RMA process (faulty i9 13900kf), the patch will have to wait until I get it back (+ the time to reinstall the devkit and the game, + the approval time for console). No ETA for the time I may get it back, unfortunately.

**Comment by o0Lil-Luna0o** [comment]
> Hello. The next patch will most likely contain some stability enhancements as well as the fix for the propagator that can't be interacted with on dedicated servers. The issue is: my computer is in an RMA process (faulty i9 13900kf), the patch will have to wait until I get it back (+ the time to reinstall the devkit and the game, + the approval time for console). No ETA for the time I may get it back, unfortunately.

Any news? With the recent update it seems that QoL might be the source of crashes on the client end. I’m going to be testing further to determine if it’s the case and will update this after.

**Comment by ItsG4T0R** [comment]
any update on this? the QOL mod is causing alot of issues with servers and needs to be updated and fixed.

**Comment by VoidArkMods** [comment]
Hello.
I got my computer back and installed the devkit again. I will install the game during the night and, if everything goes fine, I will be able to test the recent game patchnote against the next update of QoL+.
I'm expecting possible issues with the human-lookalike structures like the nanny/farmer/gardener from the patchnote, but the other features should be fine. Can you elaborate on "the mod is causing a lot of issues with servers"? The server owners I'm in touch with reported me that the mod was working great.

**Comment by o0Lil-Luna0o** [comment]
> Hello. I got my computer back and installed the devkit again. I will install the game during the night and, if everything goes fine, I will be able to test the recent game patchnote against the next update of QoL+. I'm expecting possible issues with the human-lookalike structures like the nanny/farmer/gardener from the patchnote, but the other features should be fine. Can you elaborate on "the mod is causing a lot of issues with servers"? The server owners I'm in touch with reported me that the mod was working great.

Speaking of that, I got my server running properly and letting me log on. I’ve yet to see any issues regarding QoL. I’ll keep an eye on it but so far it’s alright though I have none of the human-esk structures

**Comment by Aimhere2k** [comment]
> Hello. I got my computer back and installed the devkit again. I will install the game during the night and, if everything goes fine, I will be able to test the recent game patchnote against the next update of QoL+. I'm expecting possible issues with the human-lookalike structures like the nanny/farmer/gardener from the patchnote, but the other features should be fine. Can you elaborate on "the mod is causing a lot of issues with servers"? The server owners I'm in touch with reported me that the mod was working great.

I can say that the OoL+ Nanny is fixed in a "T-pose", with no idle animation. But at least now, since the game update, there are no longer any missing/invisible textures (previously, the Nanny was missing a third of her face, and her eyeball was visible. Which was disturbing, to say the least.)

**Comment by VoidArkMods** [comment]
Yeah, until recently the female model was using the male skeleton, which was not the case in SE/S+, causing the weird texturing. The recent changes from the previous update focused a lot on the female skeleton. I will look into it to change the pose, T-posing is not really exciting. Maybe even performing a love emote when imprinting to make it more obvious?
